tools/samtools_fastq.cwl

```diff
@@ -23,10 +23,14 @@ arguments:
               return command
           }
           
-          if(inputs.input_type == "FASTQ"){
+          if(inputs.input_type == "FASTQ" && inputs.input_reads_2 != null){
               var command =  "cp " + inputs.input_reads_1.path + " " + inputs.input_reads_1.nameroot + ".converted_1.fastq.gz && cp " + inputs.input_reads_2.path + " " + inputs.input_reads_2.nameroot + ".converted_2.fastq.gz"
               return command
           }
+          if(inputs.input_type == "FASTQ" && inputs.input_reads_2 == null){
```

Comment:
Should probably be an `else` or `else if` unless there's a way both this condition and the previous condition can be met.